### DISCUSSION TITLE
June 2023 SRD Project

### DISCUSSION THREAD

**Discussion by sigrunixia** [Obsidian-TTRPG-Community/TTRPG-Community-Admin#2] [Starry Ballot Box]
# June 2023 SRD Project Completion Vote

Voting opens on 2023-05-02 and closes on 2023-05-26. Project will be ready to work on 2023-06-01 and will be finished by 2023-06-30.

## Rules
- Other votes must be accompanied by the name of the game, and a legal notice/srd notice that indicates transcription either directly or rewording is allowed. We will not transcribe any projects where the publisher prohibits it. 

## Handicaps

These Poll Options start with extra votes as they had previously been in prior polls and were not picked. This is done to eventually assure that SRD's get completed. 

- DND 3.5e - 3
- ICRPG - 1
- PF1E - 3 (1 Vote granted due to my mistake)
- The Dark Eye - 2

### Investigating

Prowlers and Paragons - 1

## Volunteers

The more who volunteer, the less each one of us have to do to create a great experience for new players. Most SRD's are small compared to full texts we end up transcribing later on.

**Comment by ckennedy666** [comment]
It's Cepheus Engine for the Traveller derived SRD. I highly recommend sticking to the pure Cepheus Engine SRD and not using any of the variants, as Cepheus Engine is closest to Traveller.

**Reply by sigrunixia** [reply]
Ah thank you and updated!

**Comment by sigrunixia** [comment]
First "Weekend" Count. Technically I consider Friday part of my weekend. :) Also, Carl gets a vote this time just so I don't have to do manual countbacks. We got a close race already between some popular entrants. As a reminder prior entrants have their May totals as a modifier.

## Current Totals between Github, #TabletopGames, and Obsidian-TTRPG-Community
Cypher Free - 6
Cepheus Engine - 4
DnD3.5 - 3 (+3) - 6
ICRPG - 2 (+1) - 3
PF1E - 2 (+3) - 5
xWithoutNumber - 8

**Comment by sigrunixia** [comment]
Welcome to the Second Weekend Count up. XWithoutNumbers is pulling ahead of the pack with DnD3.5 not far behind. We also have our first other entry - Starfinder - Which is currently on Humble Bundle. Two weeks down, two weeks to go. Let your voice be heard. 

Votes are Tallied as (With Carlbot getting its omnipresent votes):
Obsidian TTRPG Community + Github + Obsidian TableTop Games = Pre-Total + Leftover from Prior Month(s) = Current Total 

! As a reminder: All Votes Carry Forward into future vote cycles. This is done to ensure every SRD is eventually worked even if the majority is not into the system.

## Second "Weekend" Count
DnD 3.5: 1 + 0 + 3 = 4 (+3) = 7
ICRPG: 1 + 0 + 2 = 3 (+1) = 4
PF1E: 1 + 0 + 2 = 3 (+3) = 6
The Dark Eye Wiki: 1 + 0 + 2 = 3 (+2) = 5
Cypher Free: 5 + 0 + 1 = 6
Cepheus Engine (Traveller): 2 + 0 + 2 = 4
xWithout Numbers: 5 + 1 + 5 = 11

### Other Votes
Starfinder: 2 + 0 + 0 = 2

**Comment by sigrunixia** [comment]
Happy Saturday everyone. This is the Third and Final Weekend. Poll closes at my end of day, 2023-05-26.  As we're counting down, I'll have other tally on Tuesday and then general reminders up until Friday. 

---

Votes are Tallied as (With Carlbot getting its omnipresent votes):
Obsidian TTRPG Community + Github + Obsidian TableTop Games = Pre-Total + Leftover from Prior Month(s) = Current Total

### Third Weekend Count
DnD 3.5: 1 + 0 + 3 = 4 (+3) = 7
ICRPG: 1 + 0 + 2 = 3 (+1) = 4
PF1E: 1 + 0 + 2 = 3 (+3) = 6
The Dark Eye Wiki: 1 + 0 + 2 = 3 (+2) = 5
Cypher Free: 5 + 0 + 3 = 8
Cepheus Engine (Traveller): 2 + 0 + 3 = 5
xWithout Numbers: 5 + 1 + 5 = 11

#### Other Votes

Starfinder: 2 + 0 + 1 = 3

## Housekeeping

As a reminder, Carlo has asked that for community feedback on the future of RPG Manager. Even if you have never used the plugin, please take this anonymised survey and answer in what you'd like to see in such a plugin. https://docs.google.com/forms/d/e/1FAIpQLSduhZ5WK5-jZNy8u4hYAK-442SktaRdoi15JWwic50m8A5wCg/viewform?fbzx=-1124856630504213966

**Comment by sigrunixia** [comment]
Apparently I forgot yesterday was Tuesday. :) 13th Age SRD is almost done for 1.0 release. 
A quick tally. Poll and Survey Closes Friday 11:59:59pm.

DnD 3.5: 1 + 0 + 3 = 4 (+3) = 7
ICRPG: 1 + 0 + 2 = 3 (+1) = 4
PF1E: 1 + 0 + 2 = 3 (+3) = 6
The Dark Eye Wiki: 1 + 0 + 2 = 3 (+2) = 5
Cypher Free: 5 + 0 + 3 = 8
Cepheus Engine (Traveller): 2 + 0 + 4 = 6
xWithout Numbers: 5 + 1 + 6 = 12

Other Votes

Starfinder: 2 + 0 + 1 = 3

As a reminder, Carlo has asked that for community feedback on the future of RPG Manager. Even if you have never used the plugin (There are answers that indicate as such), please take this anonymised survey and answer in the open sections what you'd like to see in such a plugin. 

docs.google.com/forms/d/e/1FAIpQLSduhZ5WK5-jZNy8u4hYAK-442SktaRdoi15JWwic50m8A5wCg/viewform?fbzx=-1124856630504213966

**Comment by sigrunixia** [comment]
## Final Tallies for June SRD

### Calculations
DnD 3.5: 1 + 0 + 3 = 4 (+3) = 7
ICRPG: 1 + 0 + 2 = 3 (+1) = 4
PF1E: 1 + 0 + 2 = 3 (+3) = 6
The Dark Eye Wiki: 1 + 0 + 2 = 3 (+2) = 5
Cypher Free: 5 + 0 + 3 = 8
Cepheus Engine (Traveller): 2 + 0 + 5 = 7
xWithout Numbers: 6 + 1 + 5 = 12
Starfinder: 2 + 0 + 1 = 3

## Post

Final Totals for June's SRD is in and xWithout Numbers is the winner with 12. All remaining entries will get their totals as extra points for July's Vote, which will be up in a few days.

Housecleaning: 
- 13th Age SRD is 99% completed for a 0.1.0 release, with some final Fantasy Statblock work to do on my end still. It will be complete by the end of the month as projected. :)
- Because Stars Without Number and Worlds Without Number are the winner, due to the increased complexity of working with these items, I am currently estimating completion will run into July. The more volunteers we have for the project, the faster it will go.
- Not inspired to do anything with the winner? feel free to take on another one. PF1E is still getting worked on, as is 13th Age, DnD 3.5, and most recently, Cypher. I'll get the repos set up for all the other entries over the weekend. 


**Totals**:
DnD 3.5e: 7
ICRPG: 4
PF1E: 6
The Dark Eye Wiki: 5
Cypher Free: 8
Cepheus Engine (Traveller): 7
xWithout Numbers: 12
Starfinder: 3
